xbmc/platform/win32/DisplayUtilsWin32.h

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Copyright (C) 2024 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "HDRStatus.h"
+
+#include <optional>
+#include <string>
+#include <vector>
+
+#include <dxgi1_5.h>
```

Comment:
not needed

Reply:
they define types used by the class functions declared in that header?

Reply:
It was needed earlier to get DISPLAYCONFIG_PATH_INFO defined without complication but doesn't seem necessary anymore. Replaced with #include <wingdi.h>.